sdk/cosmos/microsoft-azure-cosmos/src/main/java/com/azure/data/cosmos/internal/TestConfigurations.java

```diff
@@ -7,27 +7,33 @@
 
 /**
  * Contains the configurations for tests.
- * 
+ *
  * For running tests, you can pass a customized endpoint configuration in one of the following
  * ways:
  * <ul>
  * <li>-DACCOUNT_KEY="[your-key]" -ACCOUNT_HOST="[your-endpoint]" as JVM
  * command-line option.</li>
  * <li>You can set ACCOUNT_KEY and ACCOUNT_HOST as environment variables.</li>
  * </ul>
- * 
+ *
  * If none of the above is set, emulator endpoint will be used.
  */
 public final class TestConfigurations {
     // REPLACE MASTER_KEY and HOST with values from your Azure Cosmos DB account.
     // The default values are credentials of the local emulator, which are not used in any production environment.
     // <!--[SuppressMessage("Microsoft.Security", "CS002:SecretInNextLine")]-->
     public static String MASTER_KEY =
-            System.getProperty("ACCOUNT_KEY", 
+            System.getProperty("ACCOUNT_KEY",
                     StringUtils.defaultString(Strings.emptyToNull(
                             System.getenv().get("ACCOUNT_KEY")),
                             "C2y6yDjf5/R+ob0N8A7Cgv30VRDJIWEHLM+4QDU5DE2nQ9nDuVTqobD4b8mGGyPMbIZnqyMsEcaGQy67XIw/Jw=="));
 
+    public static String SECONDARY_MASTER_KEY =
+        System.getProperty("SECONDARY_ACCOUNT_KEY",
```

Comment:
The `ConfigurationManager` class in Azure Core should be used to handle loading system configurations. It will handle this logic that you have to check to system properties before the environment variables and cache it in a global store.

Reply:
@alzimmermsft thanks for the input, but for now, I will just stick to the changes made in this PR. We have plan to do these track-2 changes sometime soon. Will cover all of these in the future release, as this is kind of a hotfix out there.